molo/commenting/admin.py

```diff
@@ -15,15 +15,19 @@
 class MoloCommentAdmin(CommentsAdmin):
     list_display = (
         'comment', 'content', '_user', 'is_removed', 'is_reported',
-        'submit_date')
+        'reported_count', 'submit_date')
+    list_filter = ('submit_date', 'site', 'is_removed')
```

Comment:
This is just overwriting the list_filter specified in the parent class to remove the filter for "is_public"